villainsConvert.js

```diff
@@ -0,0 +1,92 @@
+INSERT INTO villains(name, movie, slug) VALUES(
```

Comment:
Looks like this was a temp file, these should always be deleted before committing the file version.